modules/deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode.adoc

```diff
@@ -0,0 +1,256 @@
+. Open a command-line terminal and log in to your OpenShift cluster as cluster administrator:
++
+----
+$ oc login <openshift_cluster_url> -u <admin_username> -p <password>
+----
+
+. By default, OpenShift uses a service mesh for network traffic management. As KServe raw deployment mode does not require a service mesh, disable Red Hat OpenShift Service Mesh:
+.. Enter the following command to disable Red Hat OpenShift Service Mesh:
++
+----
+$ oc edit dsci -n redhat-ods-operator
+----
+.. In the YAML editor, change the value of `managementState` for the `serviceMesh` component to `Removed`.
+.. Save the changes.
+. Create a project:
+
++
+----
+$ oc new-project <project_name> --description="<description>" --display-name="<display_name>"
+----
+For information about creating projects, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/building_applications/projects#working-with-projects[Working with projects].
+
+. Create a data science cluster:
+.. In the Red Hat OpenShift web console *Administrator* view, click *Operators* → *Installed Operators* and then click the Red Hat OpenShift AI Operator.
+.. Click the *Data Science Cluster* tab.
+.. Click the *Create DataScienceCluster* button.
+.. In the *Configure via* field, click the *YAML view* radio button.
+.. In the `spec.components` section of the YAML editor, configure the `kserve` component as shown:
++
+----
+  kserve:
+    defaultDeploymentMode: RawDeployment
+    managementState: Managed
+    serving:
+      managementState: Removed
+      name: knative-serving
+----
+.. Click *Create*.
++
+. Create a secret file: 
+.. At your command-line terminal, create a YAML file to contain your secret and add the following YAML code:
++
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    serving.kserve.io/s3-endpoint: <AWS_ENDPOINT>
+    serving.kserve.io/s3-usehttps: "1"
+    serving.kserve.io/s3-region: <AWS_REGION>
+    serving.kserve.io/s3-useanoncredential: "false"
+  name: <Secret-name>
+stringData:
+  AWS_ACCESS_KEY_ID: "<AWS_ACCESS_KEY_ID>"
+  AWS_SECRET_ACCESS_KEY: "<AWS_SECRET_ACCESS_KEY>"
+----
++
+[IMPORTANT]
+====
+If you are deploying a machine learning model in a disconnected deployment, add `serving.kserve.io/s3-verifyssl: '0'` to the `metadata.annotations` section.
+====
+.. Save the file with the file name *secret.yaml*.
+.. Apply the *secret.yaml* file:
++
+----
+$ oc apply -f secret.yaml -n <namespace>
+----
+
+. Create a service account: 
+.. Create a YAML file to contain your service account and add the following YAML code:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: models-bucket-sa
+secrets:
+- name: s3creds
+----
+For information about service accounts, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/authentication_and_authorization/understanding-and-creating-service-accounts[Understanding and creating service accounts].
```

Comment:
Hard to say without a preview but I think you need a "+" after the code block?

Reply:
As discussed, it looks like this change is not necessary.